ocs_ci/ocs/resources/mcg.py

```diff
@@ -114,6 +116,28 @@ def s3_verify_bucket_exists(self, bucket):
             logger.info(f"{bucket.name} does not exist")
             return False
 
+    def verify_s3_object_integrity(self, original_object, result_object, awscli_pod):
+        """
+        Verifies checksum between orignial object and result object on an awscli pod
+        Args:
+            orignal_object(str) : The Object that is uploaded to the s3 bucket
```

Comment:
delete space before ':'

Reply:
I offer to rename `original_object` to `original_objectname` to be more clear and concise

Reply:
changed